examples/3d/irradiance_volumes.rs

```diff
@@ -20,7 +20,8 @@ use bevy::{
     math::{uvec3, vec3},
     pbr::{ExtendedMaterial, MaterialExtension},
     prelude::*,
-    render::render_resource::{AsBindGroup, ShaderRef, ShaderType},
+    render::render_resource::{AsBindGroup, ShaderType},
```

Comment:
This might seem odd that ShaderType is coming from render, but ShaderRef is coming from shader.

Reply:
fair! i did consider this too, im not sure where i stand on whether they should be moved. ShaderType and ShaderSize are kind of encase-specific traits that we might not want in alternate renderers.